lib/u2f/register_response.rb

```diff
@@ -20,6 +21,10 @@ def self.load_from_json(json)
         raise RegistrationError, code: data['errorCode']
       end
 
+      if !data.key?('clientData') || !data.key?('registrationData')
+        raise RegistrationError, code: BAD_REQUEST
```

Comment:
The error codes are specific to the JavaScript API ([here](https://fidoalliance.org/specs/fido-u2f-v1.1-id-20160915/fido-u2f-javascript-api-v1.1-id-20160915.html#error-codes)). It would make more sense to specify an error message here than to reuse the codes from the JS API.